tests-spark310+/README.md

```diff
@@ -1,8 +1,3 @@
----
-layout: page
-title: Testing
-nav_order: 1
----
 # RAPIDS Accelerator for Testing against the upcoming version of Apache Spark
 
 While writing unit-tests, we can run into situations where we depend on classes that are only 
```

Comment:
This paragraph refers to SNAPSHOT releases which was only true when 3.1 hadn't been released.  It should just say something like "only available in specific versions of Spark."

Reply:
Removed SNAPSHOT and clarified that this folder is for tests for functionality in specific versions of Spark.